src/Tribe/Aggregator/Record/Abstract.php

```diff
@@ -2022,7 +2039,7 @@ public function insert_posts( $items = [] ) {
 			do_action( 'tribe_aggregator_after_insert_post', $event, $item, $this );
 		}
 
-		remove_filter( 'tribe-post-origin', array( Tribe__Events__Aggregator__Records::instance(), 'filter_post_origin' ), 10 );
+		remove_filter( 'tribe-post-origin', array( Records::instance(), 'filter_post_origin' ), 10 );
```

Comment:
:no_entry_sign: Short array syntax must be used to define arrays

<sub>hash: 85fbed71b7914da3d26ff62623ea8204</sub>